test/parallel/test-util-callbackify.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+const common = require('../common');
+
+const assert = require('assert');
+const { callbackify } = require('util');
+const { join } = require('path');
+const { execFile } = require('child_process');
+const fixtureDir = join(common.fixturesDir, 'uncaught-exceptions');
+const sentinalValues = [
+  'hello world',
+  null,
+  undefined,
+  { key: 'value' },
+  Symbol('I am a symbol'),
+  function ok() {},
+  ['array', 'with', 4, 'values'],
+  new Error('boo')
+];
+
+{
+  // Just works
+  for (const sentinel of sentinalValues) {
+    async function asyncFn() {
+      return await Promise.resolve(sentinel);
+    }
+
+    const cbAsyncFn = callbackify(asyncFn);
+
+    cbAsyncFn(common.mustCall((err, ret) => {
+      assert.ifError(err);
+      assert.strictEqual(ret, sentinel);
+    }));
+
+    function promiseFn() {
+      return Promise.resolve(sentinel);
+    }
+
+    const cbPromiseFn = callbackify(promiseFn);
+
+    cbPromiseFn(common.mustCall((err, ret) => {
+      assert.ifError(err);
+      assert.strictEqual(ret, sentinel);
+    }));
+  }
+}
+
+{
+  // Rejections
+  for (const sentinel of sentinalValues) {
+    async function asyncFn() {
+      return await Promise.reject(sentinel);
+    }
+
+    const cbAsyncFn = callbackify(asyncFn);
+
+    cbAsyncFn(common.mustCall((err, ret) => {
+      assert.strictEqual(ret, undefined);
+      if (err instanceof Error) {
+        assert.strictEqual(String(sentinel).endsWith(err.message), true);
+        if ('cause' in err)
+          assert.strictEqual(err.cause, sentinel);
+      } else {
+        assert.strictEqual(err, sentinel);
+      }
+    }));
+  }
+
+  for (const sentinel of sentinalValues) {
+    function promiseFn() {
+      return Promise.reject(sentinel);
+    }
+
+    const cbPromiseFn = callbackify(promiseFn);
+
+    cbPromiseFn(common.mustCall((err, ret) => {
+      assert.strictEqual(ret, undefined);
+      if (err instanceof Error) {
+        assert.strictEqual(String(sentinel).endsWith(err.message), true);
+        if ('cause' in err)
+          assert.strictEqual(err.cause, sentinel);
+      } else {
+        assert.strictEqual(err, sentinel);
+      }
+    }));
+  }
+}
+
+{
+  // 3. args work
```

Comment:
Could you make this, and the other similar comments more meaningful, or drop them completely.